config/augmentation/classification.yaml

```diff
@@ -1,31 +1,10 @@
 augmentation:
   img_size: &img_size 256
-  hsv_h: ~
-  hsv_s: ~
-  hsv_v: ~
-  degrees: ~
-  translate: ~
-  scale: ~ 
-  max_scale: ~
-  min_scale: ~  
-  crop_size_h: ~
-  crop_size_w: ~
-  resize_ratio0: ~
-  resize_ratiof: ~
-  resize_add: ~
-  shear: ~
-  perspective: ~
-  flipud: ~
-  fliplr: 0.5
-  mosaic: ~ 
-  mixup: 1.0 
-  copy_paste: ~ 
-  mixup_alpha: 0.0 
-  cutmix_alpha: 0.0 
-  mixup_switch_prob: 0.5 
-  color_jitter:
-    brightness: ~ 
-    contrast: ~ 
-    saturation: ~
-    hue: ~ 
-    colorjitter_p: ~ 
\ No newline at end of file
+  augment_info:
```

Comment:
How about just `recipe`?

Cause the root field name is `augmentation`, I believe that it doesn't need to include `augment` keyword itself.